Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/BoundingBox/BoundingBox.cs

```diff
@@ -87,7 +87,7 @@ private enum BoundsCalculationMethod
             Renderers,
             MeshFilters
         }
-        private enum BoundingBoxActivationType
+        public enum BoundingBoxActivationType
```

Comment:
Since this is public now shouldn't it get its own file?

Reply:
Agreed

Reply:
Can we just expose all these enums then?

I've exposed what I want, but I'd like anything serializable to be exposed.

The issue with making this it's own file as well is that it's only used in BoundingBox. Is there really a need to split that from the source file?

Reply:
Mainly I don't want to have the hassle of writing the `Class.EnumType` every time. 
But it's also a legibility issue and part of the coding guidelines.

Reply:
If it were possible to make the enum a bit more generic, that'd be great too

Reply:
This enum _does_ seem rather useful in more than just bounding box activations

Reply:
Well a few thoughts on that.
`Class.EnumType` helps with consistency regarding where the enum type is used. That's actually a pro to me, if it's only used in one spot.

Regarding the re-usability of the enum however, I think we could move it out once it's needed elsewhere. As in, if someone PR's a change that makes use of it elsewhere, then it should be moved to another time. Until then, I think the way it currently is setup is sufficient.

Reply:
> Regarding the re-usability of the enum however, I think we could move it out once it's needed elsewhere. As in, if someone PR's a change that makes use of it elsewhere, then it should be moved to another time. Until then, I think the way it currently is setup is sufficient.

+1

Reply:
No worries, I understand :)

Just wanted to keep the project consistent and in line with the established patterns and in line with the coding guidelines. Just have to speak up and have it noted for the record.

imho it's better to make this change now than to have a breaking change later when we decide to make that kinda change.

Reply:
Yeah that makes sense. I didn't consider it breaking code down the line. It's a tradeoff